### DISCUSSION TITLE
translation from arabic failing

### DISCUSSION THREAD

**Discussion by KrzysztofKalisiak** [Saravananslb/py-googletranslation#5] [General]
So the case is, when set to auto it returns for example Ù Ù † Ù ‰ Ø «Ø§Ù † Ù… ÙŠÙƒØ§Ù † ÙŠÙƒ / Ø¹Ø§Ù…

And when set manually src='ar': 
ValueError: invalid source language

I hope there is easy solution :)

**Comment by Saravananslb** [comment]
Hi KrzysztofKalisiak,
        Sorry for the inconvenience caused, We have updated our package.
Try our new version 1.0.4
 pip install pygoogletranslation

We are waiting for your feedback.

Thanks,
:) pygoogletranslation Team
